lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/AccessRequirementUtils.java

```diff
@@ -200,8 +222,26 @@ public static void validateAccessRequirementAcl(AccessControlList acl) {
 			ValidateArgument.required(access.getPrincipalId(), "acl.resourceAccess.principalId");
 			ValidateArgument.requirement(!BOOTSTRAP_PRINCIPAL.ANONYMOUS_USER.getPrincipalId().equals(access.getPrincipalId()), "Cannot assign permissions to the anonmous user.");
 			ValidateArgument.requirement(!BOOTSTRAP_PRINCIPAL.PUBLIC_GROUP.getPrincipalId().equals(access.getPrincipalId()), "Cannot assign permissions to the public group.");
-			Set<ACCESS_TYPE> accessSet = access.getAccessType();
-			ValidateArgument.requirement(accessSet.size() == 1 && accessSet.iterator().next() == ACCESS_TYPE.REVIEW_SUBMISSIONS, "Only the REVIEW_SUBMISSION ACCESS_TYPE is supported for access requirements.");
+		});
+	}
+
+	/**
+	 * @param acl
+	 * @param objectType
+	 * This method checks if acl has allowed access type for specific object type.
+	 * It throws IllegalArgumentException if acl contains unauthorized access type.
+	 */
+	public static void validateResourceAccessOfAclForOwnerType(AccessControlList acl, ObjectType objectType) {
+		ValidateArgument.required(acl, "acl");
+		ValidateArgument.required(objectType, "objectType");
+		acl.getResourceAccess().forEach(resourceAccess -> {
+			Set<ACCESS_TYPE> allowed_types = ALLOWED_ACCESS_TYPES.getOrDefault(objectType, new HashSet<>());
+			if (!allowed_types.isEmpty()) {
```

Comment:
Also if this is empty then no types would be allowed.  But the current code would allow all types (the opposite of what we would want).  Please add a test for this case.